cmd/frontend/auth/sign_out_cookie.go

```diff
@@ -0,0 +1,21 @@
+package auth
+
+import (
+	"net/http"
+)
+
+const SignoutCookie = "sg-signout"
```

Comment:
Now that all the logic is contained in here, you don't need to export the name of the cookie itself:

```suggestion
const signoutCookie = "sg-signout"
```

Reply:
But, not exporting it, will make it no accessible in the unit tests.